storage/store/execution_fork_evidence.go

```diff
@@ -0,0 +1,60 @@
+package store
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/storage"
+	"github.com/onflow/flow-go/storage/operation"
+)
+
+// ExecutionForkEvidence represents persistent storage for execution fork evidence.
+type ExecutionForkEvidence struct {
+	db storage.DB
+}
+
+var _ storage.ExecutionForkEvidence = (*ExecutionForkEvidence)(nil)
+
+// NewExecutionForkEvidence creates a new ExecutionForkEvidence store.
+func NewExecutionForkEvidence(db storage.DB) *ExecutionForkEvidence {
+	return &ExecutionForkEvidence{db: db}
+}
+
+// StoreIfNotExists stores the given conflictingSeals to the database.
+// This is a no-op if there is already a record in the database with the same key.
+// No errors are expected during normal operations.
```

Comment:
```suggestion
// CAUTION: This function is not synchronized and is not safe for concurrent use by multiple goroutines.
// We rely on the fact that Execution Fork Evidence has a very small surface area of use, in the Execution Fork Suppressor, for safety.
// The  Execution Fork Suppressor is responsible for ensuring mutually exclusive access to this function.
// No errors are expected during normal operations.
```

Or we could add a lock.